### DISCUSSION TITLE
Allow users to order their terms per post

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-taxonomies#1493] [Feature Requests]
Let's allow users to order terms per-post

This could be done via a setting in the metabox.

From a customer:

> I need to order terms on a per post base, eg:
> 
> – Post A
> Term one, term three, term two
> 
> – Post B
> Term three, term two, term four, term one
> 
> Is it possible with this plugin?

**Comment by stevejburge** [comment]
Related: https://twitter.com/simonrjones/status/1560208718838611969

> 
> Just discovered the term_order field in the wp_term_relationships table (where taxonomy terms are applied to a post).
> 
> But from what I can see the term_order is not set when you add a term to a post. Anyone know why or have any links to resources on this? Or is this undocumented?

**Comment by jodzeee** [comment]
Wondering if you got any further with this or if there's another solution out there. I've got a directory of musicians and would like to order their instruments by preference instead of alphabetically.

**Comment by stevejburge** [comment]
@jodzeee Sorry, not yet but it's on our roadmap
